Makefile

```
@@ -69,7 +69,7 @@ OSBASEIMAGE = gcr.io/distroless/static:nonroot
 
 SOURCE_DOCS_DIR = docs
 DEST_DOCS_DIR = content
-DOCS_GIT_REPO = https://$(DOCS_GIT_USR):$(DOCS_GIT_PSW)@github.com/crossplane/crossplane.github.io.git
+DOCS_GIT_REPO = https://$(DOCS_GIT_USR):$(DOCS_GIT_PSW)@github.com/crossplane/docs
```

Does this need to be?
```suggestion
DOCS_GIT_REPO = https://$(DOCS_GIT_USR):$(DOCS_GIT_PSW)@github.com/crossplane/docs.git
```